strawberry_django/type.py

```diff
@@ -32,9 +36,9 @@ def get_field(django_type, field_name, field_annotation=None):
     attr = get_type_attr(django_type.origin, field_name)
 
     if utils.is_field(attr):
-        field = StrawberryDjangoField.from_field(attr, django_type)
+        field = django_type.field_cls.from_field(attr, django_type)
```

Comment:
I didn't though of all possibilities, but wouldn't this break for someone not implementing his own field since if by default raises `NotImplementedError`?